tee-worker/omni-executor/executor-storage/src/pumpx_auth_token_id.rs

```diff
@@ -0,0 +1,47 @@
+use crate::{storage_key, Storage};
+use executor_primitives::Hash;
+use parity_scale_codec::{Decode, Encode};
+use rocksdb::DB;
+use std::sync::Arc;
+
+const STORAGE_NAME: &str = "pumpx_auth_token_id_storage";
+
+pub struct PumpxAuthTokenIdStorage {
+	db: Arc<DB>,
+}
+
+impl PumpxAuthTokenIdStorage {
+	pub fn new(db: Arc<DB>) -> Self {
+		Self { db }
+	}
+}
+
+impl Storage<Hash, String> for PumpxAuthTokenIdStorage {
+	fn get(&self, identity_hash: &Hash) -> Option<String> {
```

Comment:
A general question to `Storage`

shall we key it by `OmniAccount`? Imagine we have more than one identity later, so the passed-in identity might not be the "main" identity.

Just want to be generic enough and stick to the concept that omniAccount is the unique identifer for the user